packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.3
+
+- Updates Android Google maps sdk version to `18.0.2`
```

Comment:
Please follow repo CHANGELOG sytle, per the checklist. This uses the wrong list style, and is missing the period.

Reply:
fixed